client/src/pages/Medical.js

```diff
@@ -1,17 +1,109 @@
-import axios from "axios";
+// import { getAirtableRecord } from "../utils/airtable";
+// import { getLocalStorageJson } from "../utils/local-storage";
+
+// const CONTAINER_STYLES = {
```

Comment:
Maybe remove all unused code? Why keep these commented out?

Reply:
Thought, we might still want to come back to them later-on